#! /bin/sh /usr/share/dpatch/dpatch-run
## 33_asus_mycinema_remote.dpatch by Mario Limonciello <superm1@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add support for ASUS Mycinema P7131 remote (LP: #190018)

@DPATCH@
diff -urNad lirc-0.8.3~pre1~/doc/lirc.hwdb lirc-0.8.3~pre1/doc/lirc.hwdb
--- lirc-0.8.3~pre1~/doc/lirc.hwdb	2008-03-24 01:22:26.000000000 -0500
+++ lirc-0.8.3~pre1/doc/lirc.hwdb	2008-03-24 01:23:17.000000000 -0500
@@ -60,6 +60,7 @@
 Askey Magic TView CPH03x (card 1);devinput;none;hw_default;cph03x/lircd.conf.cph03x;
 Askey/Typhoon/Anubis Magic TView CPH051/061 (bt878) (card 24);devinput;none;hw_default;cph03x/lircd.conf.cph03x;
 Asus TV-Box;tvbox;lirc_dev lirc_i2c;hw_default;asus/lircd.conf.asus;
+Asus MyCinema P7131;devinput;none;hw_default;asus/mycinema.conf;
 AverMedia TV card (TVCapture, TVPhone) (card 6);devinput;none;hw_default;avermedia/lircd.conf.avermedia;
 AverMedia TV card (TVCapture98, TVPhone98) (card 13/41);devinput;none;hw_default;avermedia/lircd.conf.avermedia98;
 AverMedia TV card (VDOMATE) (use card 13);devinput;none;hw_default;avermedia/lircd.conf.vdomate;
diff -urNad lirc-0.8.3~pre1~/remotes/asus/mycinema.conf lirc-0.8.3~pre1/remotes/asus/mycinema.conf
--- lirc-0.8.3~pre1~/remotes/asus/mycinema.conf	1969-12-31 18:00:00.000000000 -0600
+++ lirc-0.8.3~pre1/remotes/asus/mycinema.conf	2008-03-24 01:22:27.000000000 -0500
@@ -0,0 +1,73 @@
+
+# Please make this file available to others
+# by sending it to <lirc@bartelmus.de>
+#
+# this config file was automatically generated
+# using lirc-0.8.2(dev/input) on Fri Jan 25 12:35:55 2008
+#
+# contributed by Gianfranco Liporace <gliporace@gmail.com>
+#
+# brand: ASUS  Remote
+# model no. of remote control: PC-39
+# devices being controlled by this remote:
+# ASUS MyCinema P7131 Hybrid Analog/DVB TV card
+
+begin remote
+
+  name  lirc.conf
+  bits           16
+  eps            30
+  aeps          100
+
+  one             0     0
+  zero            0     0
+  pre_data_bits   16
+  pre_data       0x8001
+  gap          135991
+  toggle_bit_mask 0x80010188
+
+      begin codes
+          Rec                      0x00A7
+          Close                    0x0074
+          Prev                     0x019C
+          Stop                     0x0080
+          Next                     0x0197
+          Rew                      0x00A8
+          Play/Pause               0x0077
+          Fwd                      0x00D0
+          Up                       0x0067
+          Down                     0x006C
+	  Enter                    0x8001001C
+          Left                     0x0069
+          Right                    0x006A
+          Vol+                     0x0073
+          Vol-                     0x0072
+          Ch+                      0x0192
+          Ch-                      0x0193
+          Back                     0x00AE
+          1                        0x0002
+          2                        0x0003
+          3                        0x0004
+          4                        0x0005
+          5                        0x0006
+          6                        0x0007
+          7                        0x0008
+          8                        0x0009
+          9                        0x000A
+          FullScreen               0x0174
+          0                        0x000B
+          Recall                   0x0070
+          Tv                       0x0179
+          Video                    0x0189
+          Home                     0x0066
+          Dvd                      0x0185
+          Picture                  0x016E
+          DvdMenu                  0x008B
+          Radio                    0x0181
+          Music                    0x0188
+          Mute                     0x0071
+      end codes
+
+end remote
+
+
